packages/desktop-client/src/components/reports/reports/Calendar.tsx

```diff
@@ -0,0 +1,953 @@
+                    account={undefined}
+                    transactions={transactionsGrouped}
+                    allTransactions={allTransactions}
+                    loadMoreTransactions={loadMoreTransactions}
+                    accounts={accounts}
+                    category={undefined}
+                    categoryGroups={categoryGroups}
+                    payees={payees}
+                    balances={null}
+                    showBalances={false}
+                    showReconciled={true}
+                    showCleared={false}
+                    showAccount={true}
+                    isAdding={false}
+                    isNew={() => false}
+                    isMatched={() => false}
+                    isFiltered={() => true}
+                    dateFormat={dateFormat}
+                    hideFraction={false}
+                    addNotification={addNotification}
+                    renderEmpty={() => (
+                      <View
+                        style={{
+                          color: theme.tableText,
+                          marginTop: 20,
+                          textAlign: 'center',
+                          fontStyle: 'italic',
+                        }}
+                      >
+                        No transactions
+                      </View>
+                    )}
+                    onSort={onSort}
+                    sortField={sortField}
+                    ascDesc={ascDesc}
+                    onChange={() => {}}
+                    onRefetch={() => setDirty(true)}
+                    onCloseAddTransaction={() => {}}
+                    onCreatePayee={() => {}}
+                    onApplyFilter={() => {}}
+                    onBatchDelete={() => {}}
+                    onBatchDuplicate={() => {}}
+                    onBatchLinkSchedule={() => {}}
+                    onBatchUnlinkSchedule={() => {}}
+                    onCreateRule={() => {}}
+                    onScheduleAction={() => {}}
+                    onMakeAsNonSplitTransactions={() => {}}
+                    showSelection={false}
+                    allowSplitTransaction={false}
+                  />
+                </SplitsExpandedProvider>
+              ) : (
+                <animated.div
+                  {...bind()}
+                  style={{
+                    y,
+                    touchAction: 'pan-x',
+                    backgroundColor: theme.mobileNavBackground,
+                    borderTop: `1px solid ${theme.menuBorder}`,
+                    ...styles.shadow,
+                    height: totalHeight + CHEVRON_HEIGHT,
+                    width: '100%',
+                    position: 'fixed',
+                    zIndex: 100,
+                    bottom: 0,
+                    display: isNarrowWidth ? 'flex' : 'none',
+                    flexDirection: 'column',
+                    alignItems: 'center',
+                  }}
+                >
+                  <Button
```

Comment:
LGTM! One last nitpick, Can we remove the press highlight on this button? The text disappears when the button is clicked :)

Reply:
do you mean like this?
![image](https://github.com/user-attachments/assets/3fc7cd1f-c2e1-48c5-9067-e40b1c5d5ae2)

Reply:
Yup, I think that would work :)

Reply:
I committed it already.

what's up with this action?
Compare Sizes / compare (pull_request_target) Failing after 21s
mind explaining me when it fails?

Reply:
It looked to be a flake, I've rerun and it passed now